pkgs/top-level/all-packages.nix

```diff
@@ -176,6 +176,8 @@ in
 
   buildMaven = callPackage ../build-support/build-maven.nix {};
 
+  btnx = callPackage ../tools/inputmethods/btnx {};
```

Comment:
```suggestion
  btnx = callPackage ../tools/inputmethods/btnx { };
```

Reply:
Performed the change on the next commit.